src/components/Composer/index.js

```diff
@@ -75,19 +73,20 @@ const Composer = React.forwardRef(({ className, children, sendButtonIcon, iconLe
             <Tooltip id="tooltip-attachButton" {...props}>
               {typeof (tooltipAttachButton) === 'function'
                 ? tooltipAttachButton()
-                : tooltipAttachButton
-                }
+                : tooltipAttachButton}
             </Tooltip>
           )}
         >
           <div
             {...attachButtonProps}
             className={classNames(
-              'hover:u-backgroundPrimary hover:u-textWhite u-roundedMedium u-flex u-alignItemsCenter u-justifyContentCenter u-cursorPointer'
+              'hover:u-backgroundPrimary hover:u-textWhite u-roundedMedium u-flex u-alignItemsCenter u-justifyContentCenter u-cursorPointer',
+              attachButtonProps.className && attachButtonProps.className,
```

Comment:
@thanhsonng Falsy arguments wouldn't be included in the `classNames` output. So `attachButtonProps.className` should be enough.

Reply:
@khuctrang right, this is an opportunity to improve next time!